app/views/articles/_actions.html.erb

```diff
@@ -47,7 +47,7 @@
             <button for="subscribe_notifications" id="notification-subscription-label_only_author_comments" class="notification-subscription-label" data-payload="only_author_comments">
               Post Author Comments <span class="selected-emoji">✅</span>
             </button>
-            <%= javascript_pack_tag "notificationSubscriptionHandler", defer: true %>
+            <%= javascript_packs_with_chunks_tag "notificationSubscriptionHandler", defer: true %>
```

Comment:
All `javascript_pack_tag`s have been replaced with `javascript_packs_with_chunks_tag` so that we can reference code split bundles.

Reply:
is there any further risk of duplication between chunks that have common parts of the JS included? Though IIRC that's on us to find out which parts should be common between parts of the website.

Reply:
It shouldn’t be an issue if the entry points are referenced properly. I think the issue I’m running into at the moment is that some files that were originally using separate `javascript_pack_tag ` need to be consolidated into one `javascript_packs_with_chunks_tag`. Still investigating.